lib/build/amodro-trace/write/stubs.js

```diff
@@ -12,7 +13,9 @@ module.exports = function stubs(options) {
   options = options || {};
 
   return function(context, moduleName, filePath, contents) {
-    if (options.stubModules && options.stubModules.indexOf(moduleName) !== -1) {
+    if (options.stubModules
```

Comment:
Yes I wasn't sure whether that was necessary....this change was made to allow the plugin to be defined as a package in the bundle dependencies - see [the config for systemjs-plugin-text](https://github.com/aurelia/cli/pull/631/commits/a818cd8591daf5ac73908f6b108ddbf4e53e8bb2#diff-239f8d9c8592bbc58e040901f2100f2bR8)